core/src/main/java/com/amplifyframework/predictions/models/SpeechType.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.predictions.models;
+
+/**
+ * Different types of parts of speech.
+ */
+@SuppressWarnings("JavadocVariable")
+public enum SpeechType {
+    ADJECTIVE,
```

Comment:
Can we get javadocs on all of these? Some of them, like `ADVERB`, are already clear. However, there are some that will be less familiar to the average user, and some that are just ambigious. For example, I don't know what a `DETERMINER` or a `COORDINATING_CONJUNCTION` are, in the natural language domain. But, a simple text above them with an example of one of the things would go a long way.

For example,,
```
/**
 * Modifies or describes a noun.
 * For example, in the phrase "A fast car", "fast" is an adjective which modifies the noun "car."
 */
ADJECTIVE,
```